src/pages/trails/ballots/components/BallotForm.vue

```diff
@@ -85,6 +85,8 @@ export default {
       return this.form.treasurySymbol?.symbol !== "VOTE" && this.isStakeable;
     },
     available() {
+      // TODO: borrame
+      console.log("BallotForm.computed.available -> this.userBalance: ", this.userBalance);
```

Comment:
I guess we can delete this 🤔

Reply:
[f6280a2](https://github.com/telosnetwork/app-telos-native/pull/125/commits/f6280a25e41c0e182dd94fbddd14dbb7d0865993)